jiant/tasks/qa.py

```diff
@@ -362,7 +365,7 @@ def _make_instance(psg, qst, ans_str, label, psg_idx, qst_idx, ans_idx):
 
             ent_strs = example["ents"]
             ents = [
-                process_sentence(self._tokenizer_name, ent, self.max_seq_len)[1:-1]
+                truncate_and_tokenize(self._tokenizer_name, ent, self.max_seq_len)[1:-1]
```

Comment:
Does `truncate_and_tokenize` still append SOS/EOS tokens? If not, this is likely the source of the bug as you're throwing away lots of the entities spans.

Reply:
Thanks! Yes, this does look fishy...

Reply:
Fixed, though I'm still investigating lowish performance on ReCoRD...